app/controllers/dashboard/groups_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Dashboard
+  # Dashboard groups controller
+  class GroupsController < ApplicationController
+    def index
+      respond_to do |format|
+        format.html do
+          @groups = authorized_scope(Group, type: :relation).without_descendants.include_route.order(updated_at: :desc)
+        end
+        format.turbo_stream do
+          @collapsed = params[:collapse] == 'true'
+          @group = Group.find(params[:parent_id])
+          @depth = params[:depth].to_i
+          @sub_groups = @group.children.first(10)
```

Comment:
**nitpick (blocking):** For now can you remove the `first` since in a subsequent PR we will only load the first X and then have the view more link. This just makes it so that all subgroups can be found with this current implementation.

Reply:
Crap sorry I missed that one.  Remove in d694ca4